en/docs/reference/mediators/enrich-mediator.md

```diff
@@ -217,7 +248,7 @@ Below is the JSON payload that is sent in the request for the following examples
 }
 ```
 
-### Example 1: Extract content from the message payload and set it to the message body
+#### Example 1: Extract content from the message payload and set it to the message body
 
 In this example, we will extract the content in the `data` object and set it as the message body.
```

Comment:
```suggestion
In this example, you'll extract the content in the `data` object and set it as the message body.
```

Let's replace `we will` with `you'll`. There are multiple occurrences within the page.